index.html

```diff
@@ -317,7 +305,7 @@ <h2>Glossary</h2>
 
 <p><dfn data-lt="metadata|metadata" id="def_metadata" class="lint-ignore">Metadata</dfn> is additional information about data. Key types of metadata for <a class="termref" href="#internationalization">internationalization</a> are <a class="termref" href="#def_language_metadata">language metadata</a> and metadata to support <a class="termref" href="#def_bidirectional_text">bidirectional text</a>. Metadata has a scope, e.g., a string or a set of strings. In absence of explicit metadata, defaults might apply, e.g. defaults for the <a class="termref" href="#def_base_direction">base direction</a> of a text.</p>
 
-
+<p><dfn id="def_mojibake" class="lint-ignore">Mojibake</dfn> (文字化け). Garbled or incorrectly rendered or processed characters, generally caused by using the wrong <a>character encoding</a> to interpret the bytes in a string or file. The word is Japanese in origin and is pronounced <q>/mo.d&#x0361;&#x0292;i.ba.ke/</q>. For example, the word <q>&#x6587;&#x5B57;&#x5316;&#x3051;</q> encoded as UTF-8 might be displayed as <q>&#x00e6;&#x0096;&#x0087;&#x00e5;&#x00ad;&#x0097;&#x00e5;&#x008c;&#x0096;&#x00e3;&#x0081;&#x0091;</q> if viewed in an application that thinks (incorrectly) that the character encoding is <code>ISO-8859-1</code>.</p>
```

Comment:
- how about to add lang=ja to characters not in English?
- how about to mention on U+FFFD, which is often shown for cases of non-Unicode two byte encoding in UTF-8?
- (just for comment, I believe no well confirmed standard there...) [wikipedia じ](https://ja.wikipedia.org/wiki/%E3%81%98) uses U+291 instead of U+292 (and in some Japanese dictionary, just `/ʒi/` within a word...)